docs/docs/configuration/face_recognition.md

```diff
@@ -26,7 +26,7 @@ In both cases, a lightweight face landmark detection model is also used to align
 
 The `small` model is optimized for efficiency and runs on the CPU, most CPUs should run the model efficiently.
 
-The `large` model is optimized for accuracy, an integrated or discrete GPU is highly recommended.
+The `large` model is optimized for accuracy, an integrated or discrete GPU is highly recommended. Intel users should use the default Docker image to run on an iGPU with OpenVINO and Nvidia users should use the `-tensorrt` Docker image to run on dedicated Nvidia GPUs. Face recognition does not run on a Google Coral.
```

Comment:
this also works on AMD. Realistically I think we probably want to follow the pattern of the note on the semantic search docs

Reply:
Okay, wasn't sure so I left it out.

Maybe it'd be better to find a central place for that note in the Semantic Search section so that we can just link to it from there and all the other enrichments.